lib/charms/glauth_k8s/v0/ldap.py

```diff
@@ -284,8 +285,10 @@ def remove(self) -> None:
 
 class LdapProviderBaseData(BaseModel):
     urls: List[str] = Field(frozen=True)
+    ldaps_urls: List[str] = Field(frozen=True)
     base_dn: str = Field(frozen=True)
     starttls: StrictBool = Field(frozen=True)
+    ldap_over_tls: StrictBool = Field(frozen=True)
```

Comment:
Why is this needed? The requirer charm should be able to infer whether ldaps is enabled by checking if `ldaps_urls` is populated.

Reply:
You are right, it's not needed.
I added it in mirror of `starttls_enabled` but it can be removed, I'll do it.